components/BlogPost/index.tsx

```diff
@@ -0,0 +1,33 @@
+import MarkdownRenderer from '../MarkdownRenderer';
```

Comment:
Move this component to `landing/blog` for now. However, try to coordinate with @feliciofilipe the best way to create a `/blog` folder that will have everything related with the blog and better split the components into areas.